submissions/Levi123/tiny-JS/index.js

```diff
@@ -0,0 +1,81 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Levi123/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Rich',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Hav-hav!',
+   friends: ['Bonya'],
+}
+
+const cat = {
+   species: 'cat',
+   name: 'Bonya',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'Mr-mr-mr',
+   friends: ['Rich', 'Catwoman'],
+}
+
+const female = {
+   species: 'human',
+   name: 'Olya',
+   gender: 'female',
+   legs: 4,
+   hands: 4,
+   saying: "Glory to Ukraine",
+   friends: ['Yura', 'Rich'],
+}
+
+const male = {
+   species: 'human',
+   name: 'Yura',
+   gender: 'male',
+   legs: 4,
+   hands: 4,
+   saying: "Glory to Ukraine",
+   friends: ['Olya', 'Bonya'],
+}
+
+const catwoman = {
+   species: 'human',
+   name: 'Catwoman',
+   gender: 'female',
+   legs: 4,
+   hands: 4,
+   saying: cat.saying,
+   friends: ['Bonya']
+}
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+function printObject(object){
+   const prop = [];
+   for (element in object){
+      prop.push(object[element]);
+   }
+   let newStr = prop.join('; ');
+   print(newStr);
+}
```

Comment:
> - Give variables and functions meaningful names. Avoid generic names like `item`, `element`, `key`, `object`, `array` or their variations. Exception: helper functions that are specifically and intentionally designed to be multipurpose.

> - [ ] `Object` methods like `keys`, `values`, `entries` shouldn't be used when a particular order is required as these do not guarantee any particular order of keys/values. Same refers to `for...of` and `for...in` when applied to objects.

Reply:
> > * Give variables and functions meaningful names. Avoid generic names like `item`, `element`, `key`, `object`, `array` or their variations. Exception: helper functions that are specifically and intentionally designed to be multipurpose.
> 
> > * [ ]  `Object` methods like `keys`, `values`, `entries` shouldn't be used when a particular order is required as these do not guarantee any particular order of keys/values. Same refers to `for...of` and `for...in` when applied to objects.

I don't understand second point of your massage, i can't use (for...in) at all? and why?

Reply:
Q: I can't use (for...in) at all? and why?

A: As per explanation above "...when a particular order is required as these do not guarantee any particular order of keys/values."

Only arrays guarantee order of elements.

As a peer developer, knowing that in objects order of keys is not important I can define new objects with properties in different order. It would be great if this didn't affect the code.

Reply:
> Q: I can't use (for...in) at all? and why?
> 
> A: As per explanation above "...when a particular order is required as these do not guarantee any particular order of keys/values."
> 
> Only arrays guarantee order of elements.
> 
> As a peer developer, knowing that in objects order of keys is not important I can define new objects with properties in different order. It would be great if this didn't affect the code.

I try to change my errors.